src/main/java/org/zalando/nakadi/domain/SubscriptionListWrapper.java

```diff
@@ -0,0 +1,19 @@
+package org.zalando.nakadi.domain;
+
+import javax.annotation.concurrent.Immutable;
+import java.util.Collections;
+import java.util.List;
+
+@Immutable
+public class SubscriptionListWrapper {
+
+    private final List<Subscription> data;
+
+    public SubscriptionListWrapper(final List<Subscription> data) {
+        this.data = data;
+    }
+
+    public List<Subscription> getData() {
```

Comment:
I think we should agree on either "data" or "items". I prefer "items" since data doesn't evoke for a list.